aimmo-game-worker/simulation/world_map.py

```diff
@@ -51,7 +53,10 @@ def score_cells(self):
         return (c for c in self.all_cells() if c.generates_score)
 
     def pickup_cells(self):
-        return (c for c in self.all_cells() if c.pickup)
+        return (c for c in self.all_cells() if getattr(self, 'pickup', False))
+
+    def partially_fogged_cells(self):
+        return (c for c in self.all_cells() if c.partially_fogged)
 
     def is_on_map(self, location):
         return location in self.cells
```

Comment:
This now checks if the cell is in known/partially fogged (ie fully fogged cells give False). If we're not revealing map size then we should rename this method.